frontend/src/components/VerticalNavigation.js

```diff
@@ -43,7 +43,7 @@ const VerticalNavigation = () => {
           {/* <li><img src={Friends} className='Friendspic' alt='Friends' /><a href="/friends">FRIENDS</a></li> */}
```

Comment:
remove it if not necessary

Reply:
I am going to leave it since we need that once friends page is fully implemented